spec/mysql2/client_spec.rb

```diff
@@ -136,25 +131,34 @@ def connect *args
         :sslcapath => '/etc/mysql/',
         :sslcipher => 'DHE-RSA-AES256-SHA'
       )
-    }.should_not raise_error(Mysql2::Error)
+    }.not_to raise_error
 
     results = ssl_client.query("SHOW STATUS WHERE Variable_name = \"Ssl_version\" OR Variable_name = \"Ssl_cipher\"").to_a
-    results[0]['Variable_name'].should eql('Ssl_cipher')
-    results[0]['Value'].should_not be_nil
-    results[0]['Value'].should be_kind_of(String)
-    results[0]['Value'].should_not be_empty
+    expect(results[0]['Variable_name']).to eql('Ssl_cipher')
+    expect(results[0]['Value']).not_to be_nil
+    expect(results[0]['Value']).to be_kind_of(String)
+    expect(results[0]['Value']).not_to be_empty
 
-    results[1]['Variable_name'].should eql('Ssl_version')
-    results[1]['Value'].should_not be_nil
-    results[1]['Value'].should be_kind_of(String)
-    results[1]['Value'].should_not be_empty
+    expect(results[1]['Variable_name']).to eql('Ssl_version')
+    expect(results[1]['Value']).not_to be_nil
+    expect(results[1]['Value']).to be_kind_of(String)
+    expect(results[1]['Value']).not_to be_empty
 
     ssl_client.close
   end
 
+  def run_gc
+    if defined?(Rubinius)
+      GC.run(true)
```

Comment:
Oh, does this force Rubinius GC immediately even without the environment variable?